lib/action_network_rest/client.rb

```diff
@@ -32,7 +32,23 @@ def people
     end
 
     def petitions(petition_id=nil)
```

Comment:
Should we DRY this pattern?

Reply:
Hmm, not sure -- there's definitely some common structure, but I'm not sure how doable it is to DRY without sacrificing readability.

Reply:
👍